dlk/python/dlk/templates/include/quantizer.h

```diff
@@ -53,25 +53,17 @@ void func_QTZ_binary_channel_wise_mean_scaling(
 
   for (unsigned i = 0; i < in_channel; i++) {
     sum[i] = 0;
-    mean[i] = 0;
   }
 
-  for(unsigned i = 0; i < in_channel; i++)
-  for(unsigned j = 0; j < num_elems_in_channel; j++)
-  {
-    sum[i] += std::abs(input[i * num_elems_in_channel + j]);
-  }
-
-  for(unsigned i = 0; i < in_channel; i++)
-  {
+  for(unsigned i = 0; i < in_channel; i++) {
+    for(unsigned j = 0; j < num_elems_in_channel; j++) {
+      sum[i] += std::abs(input[i * num_elems_in_channel + j]);
+    }
```

Comment:
does `sum` still has to be array now?

it seems we could just rewrite it like
```cpp
for(unsigned i = 0; i < in_channel; i++) {
  T_FLOAT sum = 0;
  for(unsigned j = 0; j < num_elems_in_channel; j++) {
    sum += std::abs(input[i * num_elems_in_channel + j]);
  }

  mean[i] = sum / num_elems_in_channel;
...
```
without any semantic loss, as `sum` is not used anywhere else... :thinking:

Reply:
Thanks to you, we can remove two array. Thank you n-nez san.